test/domain/questions_test.py

```diff
@@ -0,0 +1,33 @@
+"""
+Tests for domain methods in questions.py
+"""
+import mock
+
+from app.domain.questions import get_survey_page
+from test.fixtures.appengine import GaeTestCase
+
+
+class GetSurveyPageTests(GaeTestCase):
+
+    def test_page_num_is_required(self):
+        with self.assertRaises(ValueError):
+            get_survey_page(None)
+
+    def test_page_num_must_be_positive_integer(self):
+        with self.assertRaises(ValueError):
+            get_survey_page(-1)
+
+    @mock.patch('app.models.question.Question.get_questions_by_number_range')
```

Comment:
So yeah @gholtslander I was able to get this working eventually. I now can do this instead of importing all 180 questions for every test (which was terrible). The tests now run in a few hundred milliseconds (after a second or so of bootstrapping) as opposed to five or so seconds. Woo!